CMakeLists.txt

```diff
@@ -24,25 +24,35 @@ if (UNIX AND NOT APPLE AND NOT ANDROID AND NOT WEBGL)
 endif()
 
 if (WIN32)
+    option(USE_STATIC_CRT "Link against the static runtime libraries." ON)
+
     # On Windows we need to instruct cmake to generate the .def in order to get the .lib required
     # when linking against dlls. CL.EXE will not generate .lib without .def file (or without pragma
     # __declspec(dllexport) in front of each functions).
     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
+    if (${USE_STATIC_CRT})
+        set(_CRT_FLAGS_RELEASE "/MT")
```

Comment:
Don't prefix the variable name with `_`

Reply:
Yup I was trying to signify a "local" variable, heh. Fixed it.